.github/ISSUE_TEMPLATE/issue-template.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Issue Template
+about: Template for issues
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- If you're describing a bug, tell us what should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+<!--- or ideas how to implement the addition or change -->
+
+## Steps to Reproduce (for bugs)
+<!--- Provide a link to a live example, or - -->
+<!--- Fork the CodePen to reproduce https://codesandbox.io/s/fixed-data-table-example-3gd9x ->
```

Comment:
Updated code sandbox link.

nit: change `CodePen` to `CodeSandBox`

Reply:
I don't have a problem with making that change.  Maybe we could have both a simple and an advanced sandbox example?